CHANGES.next.md

```diff
@@ -1,5 +1,6 @@
 ### Breaking changes:
 
+-   Add ibmcloud as a new provider
```

Comment:
Please move this to `New Features`. This should not be a breaking change.

Reply:
yes, I moved it, thanks.